weather-app/src/app/app.module.ts

```diff
@@ -8,6 +8,8 @@ import {HeaderComponent} from "./header/header.component";
 import { HomeComponent } from './home/home.component';
 import { DetailsComponent } from './details/details.component';
 import { ForecastComponent } from './forecast/forecast.component';
+import { DetailsTableComponent } from './details/details-table/details-table.component';
+import { DetailsBtnsComponent } from './details/details-btns/details-btns.component';
```

Comment:
this components must be in details folder

Reply:
Thank you so much. Resolved